server/src/main/resources/file.conf

```diff
@@ -29,6 +29,8 @@ service {
   enableDegrade = false
   #disable
   disable = false
+  max.commit.retry.timeout = "-1"
```

Comment:
give example . because you use many unit such as m,s. Maybe they'll use min.

Reply:
Yes, you are right.I added comments here to help users understand how to configure.

Reply:
what is -1